Code/Framework/AzCore/AzCore/Serialization/Json/JsonMerger.cpp

```diff
@@ -285,7 +285,7 @@ namespace AZ
         }
 
         rapidjson::Value newValue;
-        newValue.CopyFrom(value->value, allocator);
+        newValue.CopyFrom(value->value, allocator, true);
```

Comment:
Suggestion: Consider using a constexpr bool X = true for these CopyFrom calls so that the parameter has a descriptive name. Just reading it in the context of the PR, i can't explain what this parameter is or why its presence is good.

Reply:
I'll do that.
The `rapidjson:Value::CopyFrom` doesn't actually copy const strings by default which causes dangling string references when the source `rapidjson::Value` being copied from goes out of scope.